### PR TITLE
Created contacts listing page UI

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by allenjiji**
Fixes #4 
Also made changes as per the review suggestions of PRs #9 and #8 
image

@yedhink 
Please review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    dataIndex: "menu",
+    key: "menu",
+    width: 150,
+    render: () => (
+      <div>
+        <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
+          <li> Edit </li>
+          <li> Delete </li>
+        </Dropdown>
+      </div>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.jsx:48]: Why's this `div` required here?

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+import { Button, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS_TABLE_COLUMNS, CONTACTS_TABLE_ROWS } from "./constants.jsx";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:6]: Avoid file name extensions. Webpack can automatically resolve extensions: https://webpack.js.org/configuration/resolve/#resolveextensions

Also read: https://stackoverflow.com/a/55289496

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+const Contacts = () => {
+  const [showMenu, setShowMenu] = useState(true);
+
+  return (
+    <>
+      <SideMenuBar showMenu={showMenu} title="Contacts" />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(!showMenu)}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:18]: Go through the Slack thread very carefully.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 4]
```diff
@@ -54,24 +54,22 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
...
             <Select
-              defaultMenuIsClose
               isMulti
               required
               className="w-full flex-grow-0"
               label="Tags"
               name="tags"
               options={TAGS_DROPDOWN_DATA}
-              placeholder="Select Role"
+              placeholder="Select Tags"
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:72]: Maintain some consistency. Stick onto Sentence case for the messages shown in UI. Your placeholders follow different casing right now.

**Review by yedhink — COMMENTED** [review]
@allenjiji

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 3 above)
Comment (allenjiji) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx:18]: Sorry, I've gone through the slack thread again but I couldn't get the mistake in these lines.
Can you please point it out @yedhink ?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx:18]: @allenjiji https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

**Pull request merged**